docs/db-optimization-verification.html

```
@@ -0,0 +1,978 @@
+      </div>
+
+    </div>
+  </div>
+
+  <!-- SECTION 7: PERFORMANCE -->
+  <div class="section">
+    <div class="section-header">
+      <div class="section-num">7</div>
+      <div>
+        <h2>Замер производительности</h2>
+        <div class="section-desc">Убедиться, что p95 latency < 200ms (NFR)</div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)